### ISSUE TITLE
[4.17.16] Uncaught ReferenceError: _ is not defined (Webpacker, Vue)

### ISSUE THREAD

**Issue by afuno** [lodash/lodash#4844]
The problem arose in the latest version. Setting specific version (4.17.15) in package.json fixes this problem. At the same time, version 4.17.16 will still be available for third-party libraries with dependencies. That is, the problem is with Webpacker and/or Vue.

In a Vue application, I integrate Lodash as follows:

```js
import Vue from 'vue'
import Lodash from 'lodash'
Vue.use(Lodash)
```

**Comment by victorpolko** [comment]
I guess the issue was with the version `4.17.17` which should be a minor/major version change, because the structure of the files was changed drastically.

We also experienced a couple of builds failing due to this issue, even though we use `﻿﻿"lodash": "^4.17.15",` in the `package.json`.

Guys have already removed this wrong version, and the latest now is `4.17.16`, so I guess it's fine to stick to the old methods. But I won't.
